src/jit/rationalize.cpp

```diff
@@ -157,19 +157,20 @@ void Compiler::fgFixupArgTabEntryPtr(GenTreePtr parentCall, GenTreePtr oldArg, G
     }
 }
 
-// Rewrite InitBlk involving SIMD vector into stlcl.var of a SIMD type.
+// Rewrite a SIMD indirection as GT_IND(GT_LEA(obj.op1)), or as a simple
+// lclVar if possible.
 //
 // Arguments:
-//    ppTree      - A pointer-to-a-pointer for the GT_INITBLK
+//    ppTree      - A pointer-to-a-pointer for the GT_BLK or GT_OBJ
 //    fgWalkData  - A pointer to tree walk data providing the context
```

Comment:
Ah yes, though that wasn't me ;-) I'll update the header comment.

---
In reply to: [77410463](https://github.com/dotnet/coreclr/pull/7027#discussion_r77410463) [](ancestors = 77410463)